headless-server-commerce-app/src/main/java/com/coremedia/blueprint/boot/headlessserver/HeadlessServerCommerceApp.java

```diff
@@ -43,6 +40,8 @@ public static void main(String[] args) {
         return;
       }
       LOG.error("Application startup failed, cause: {}", e.getMessage());
+    } catch (Exception e) {
+      LOG.error("Application startup failed, cause: {}", e.getMessage());
```

Comment:
Could we generalize the other catchphrase to `Exception` and that way merge these 2 catches?
I stumbled upon this, because right now, L. 44 duplicates L. 42.